en/docs/api-manager/4.5.0/assets/css/theme-material.css

```
@@ -0,0 +1,1741 @@
+   margin:0;
+   background: #ffffff;
+   font-family: "Roboto","Helvetica","Arial",sans-serif;
+ }
+
+ .swagger-ui .opblock {
+     margin: 0 0 15px;
+     border: none;
+     border-radius: 2px;
+     box-shadow: 0 2px 2px 0 rgba(0,0,0,.14),0 3px 1px -2px rgba(0,0,0,.2),0 1px 5px 0 rgba(0,0,0,.12);
+ }
+© 2019 GitHub, Inc.
```

Stylelint (17.14.0)

[error] Unknown word © (CssSyntaxError)

(CssSyntaxError)